README.md

```diff
@@ -429,6 +432,30 @@ export default MyApp;
 
 This will cache the `messages.list` so it's instant when a user visits the page.
 
+
+### Invoking route directly
+
+You can also invoke a route directly and pass the data as props.
+
+
+<details><summary>In `getStaticProps`</summary>
+
+```tsx
+import { appRouter } from './api/trpc/[...trpc]'; // Important - only ever import & use this in the SSR-methods
+
+export async function getStaticProps() {
+  const allPosts = await appRouter.createCaller({}).query('allPosts', { limit: 100 })
+
+  return {
+    props: {
+      allPosts: trpc.dehydrate(),
+    },
+    revalidate: 1,
+  };
+}
+```
+</details>
+
```

Comment:
@cyrus-za I've added you as a maintainer so you should be able to push to this branch - could you add more details here? I guess one would want the `inferServerSideProps` or whatever it's called from next

Reply:
You mean `InferGetStaticPropsType<typeof getStaticProps>` ?

Reply:
that